lib/trino-parquet/src/main/java/io/trino/parquet/reader/MetadataReader.java

```diff
@@ -294,7 +293,7 @@ public Optional<Boolean> visit(LogicalTypeAnnotation.StringLogicalTypeAnnotation
                 .orElse(FALSE);
     }
 
-    private static void tryReadOldUtf8Stats(Statistics statistics, BinaryStatistics columnStatistics)
+    private static org.apache.parquet.column.statistics.Statistics<?> tryReadOldUtf8Stats(Statistics statistics, org.apache.parquet.column.statistics.Statistics<?> columnStatistics)
```

Comment:
Let's retain `BinaryStatistics` as the method argument type, makes this method less likely to be misused